gui/api/resources.py

```diff
@@ -3992,7 +3995,8 @@ def dehydrate(self, bundle):
                     '_vncweb_url': device_vncweb_url
                 })
             if bundle.obj.device_set.filter(dtype='VNC').exists():
-                vnc_port = bundle.obj.device_set.filter(dtype='VNC').values_list('attributes', flat=True)[0].get('vnc_port', 5900 + bundle.obj.id)
+                vnc_port = bundle.obj.device_set.filter(
+                    dtype='VNC').values_list('attributes', flat=True)[0].get('vnc_port', 5900 + bundle.obj.id)
```

Comment:
This will not be used in 12, please revert changes here to reduce the diff between master.